package.json

```diff
@@ -26,7 +26,6 @@
   "packageManager": "yarn@3.3.1",
   "private": true,
   "scripts": {
-    "lint": "lerna run lint --concurrency 4",
```

Comment:
Why remove this?

Reply:
I added that line in the branch this PR is set to merge into. This diff will disappear once https://github.com/heroku/cli/pull/2214 is merged.
I deleted it because I didn't think `yarn lint` was concise enough over `yarn lerna run lint` to justify putting it in the root scripts. I'm indifferent and can easily add it back.